src/manifest.ts

```diff
@@ -1,26 +1,44 @@
+import { accessSync } from 'fs'
 import { readFile } from 'fs-extra'
 import * as path from 'path'
 import { memoize } from 'ramda'
 
 import { CommandError } from './errors'
 
 const readFileUtf = async (file: string): Promise<string> => {
-  try {
-    return await readFile(file, 'utf8')
-  } catch (e) {
-    throw new CommandError(`Manifest file doesn't exist or is not readable. Please add a manifest.json file in the root of the app folder.`)
+  return await readFile(file, 'utf8')
+}
+
+export const manifestFileName = 'manifest.json'
+
+export const getAppRoot = () => {
+  const cwd = process.cwd()
+
+  const find = dir => {
+    try {
+      accessSync(path.join(dir, manifestFileName))
+      return dir
+    } catch (e) {
+      if (dir === '/') {
```

Comment:
does this work on windows? (i.e. is the root directory `/`?)

Reply:
You can use https://nodejs.org/api/path.html#path_path_sep

Reply:
@klzns I don't think that works, because on windows the root would be something like `C:\\`, I could use [this](https://nodejs.org/api/path.html#path_path_parse_path) to get the root dir name, what do you think?

Reply:
Hey, check my comment on the pull request.

If we set just a specific number of levels that we go up, that wouldn't be an issue.